benchmarks/bge_m3/src/scripts/bench_onnx_cpu_fp32.py

```
@@ -147,6 +158,72 @@ def main() -> None:
+            truncation=True,
+            max_length=args.max_length,
+            return_tensors="np",
+        )
+        inputs_val = make_inputs(session, encoded_val)
+
+        outputs_ref = session.run(None, inputs_val)
+        output_names = [o.name for o in session.get_outputs()]
+        output_shapes = [list(o.shape) for o in outputs_ref]
+        attention_mask_val = encoded_val["attention_mask"]
+
+        ref_embeddings, extraction_method_ref = extract_embeddings(
```

Ruff (0.15.14)

[warning] Unpacked variable `extraction_method_ref` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)